vue-app/src/views/AboutHowItWorks.vue

```diff
@@ -148,7 +148,12 @@ import Links from '@/components/Links.vue'
 
 // TODO connect with state to add durations to content
```

Comment:
✂️

Reply:
In addition - we should pull in those time variables. See the "TODO" comments in this file. e.g. "This phase will last xxx days" & "There will be a maximum of 125 projects in the round" - those amounts should both be variables we pull in dynamically.